velox/exec/Spiller.cpp

```diff
@@ -547,6 +568,33 @@ void Spiller::finalizeSpill() {
   finalized_ = true;
 }
 
+void Spiller::fillSpillRuns(
+    std::vector<char*>& rows,
+    int32_t numRows,
+    std::vector<uint64_t>& hashes,
+    bool isSinglePartition) {
+  // Calculate hashes for this batch of spill candidates.
+  auto rowSet = folly::Range<char**>(rows.data(), numRows);
```

Comment:
Can we just pass rowSet to this function?

Reply:
This function would be used in a for loop, pass `hashes` and `isSinglePartiton` calculated outside the loop.